exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -322,6 +332,25 @@ func (b *Bitfinex) UpdateTradablePairs(ctx context.Context, forceUpdate bool) er
 	return b.EnsureOnePairEnabled()
 }
 
+// UpdateOrderExecutionLimits sets exchange execution order limits for an asset type
+func (b *Bitfinex) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return common.ErrNotYetImplemented
+	}
+	pairs, err := b.GetSiteInfoConfigData(ctx, a)
+	if err != nil {
+		return err
+	}
+	limits := make([]order.MinMaxLevel, 0, len(pairs))
+	for v := range pairs {
```

Comment:
Can use instead so we don't need a double map lookup. Also could return just a slice instead of a map if you want. Up to you.
```go
for _, v := range pairs {
```

Reply:
Good point. We'll keep the map though.